src/main/java/edu/cornell/kfs/sys/service/impl/DisposableClientServiceImplBase.java

```diff
@@ -55,5 +63,19 @@ protected void closeClientQuietly() {
         
         CURestClientUtils.closeQuietly(jerseyClient);
     }
+    
+    protected void disableRequestChunkingIfNecessary(Client client, Invocation.Builder requestBuilder) {
```

Comment:
Now that you've moved this method to the superclass, the `client` arg is masking the `client` instance variable. You may want to make adjustments to avoid confusion, such as renaming the arg or making the method a static one.